src/transactions/types.ts

```diff
@@ -226,15 +225,15 @@ export type InputSimulateTransactionData = {
   /**
    * For a single signer transaction
    */
-  signerPublicKey: PublicKey;
+  signerPublicKey: PublicKeyInput;
   /**
    * For a fee payer or multi-agent transaction that requires additional signers in
    */
-  secondarySignersPublicKeys?: Array<PublicKey>;
+  secondarySignersPublicKeys?: PublicKeyInput[];
```

Comment:
as mentioned in the previous PR - there is no firm reason this syntax was chosen, but since we already have it throughout the SDK we might as well keep going with it

```suggestion
  secondarySignersPublicKeys?: Array<PublicKeyInput>;
```

Reply:
ok sure